data/json/monsters/feral_humans.json

```
@@ -58,6 +58,55 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_feral_sapien_spear",
+    "type": "MONSTER",
+    "name": { "str": "feral sapien" },
+    "description": "In this feral's eyes is more intelligence than in most of their kind.  They appear to be holding a spear.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "special_attacks": [
+      {
+        "id": "stretch_attack",
+        "cooldown": 10,
+        "range": 2,
+        "hit_dmg_u": "The %1$s thrusts a spear at your %2$s, slashing it!",
+        "hit_dmg_npc": "The %1$s thrusts a spear at <npcname>'s %2$s, slashing it!",
+        "no_dmg_msg_u": "The %1$s thrusts a spear at your %2$s, but glances off your armor!",
+        "no_dmg_msg_npc": "The %1$s thrusts a spear at <npcname>'s %2$s, but glances off armor!",
+        "miss_msg_u": "The %s thrusts a spear at you, but you evade it!",
+        "miss_msg_npc": "The %s thrusts a spear at <npcname>, but they evade it!"
+      }
+    ],
+    "armor_acid": 4,
+    "harvest": "human",
+    "vision_day": 50,
+    "vision_night": 3,
+    "path_settings": { "max_dist": 30, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "starting_ammo": { "rock": 3 },
+    "death_drops": "feral_sapien_death_drops_spear",
+    "zombify_into": "mon_zombie_survivor",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "HUMAN", "CAN_OPEN_DOORS", "PATH_AVOID_DANGER_1" ]
```

```suggestion
    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "HUMAN", "CAN_OPEN_DOORS", "PATH_AVOID_DANGER_1", "PRIORITIZE_TARGETS" ]
```
This may require running through the JSON "linter" given the length of the line. (It results in their being at least _slightly_ smarter in targeting, to go along with the increased `max_dist`.)